### PR TITLE
feat: context handler for titles

### PR DESCRIPTION
**Pull request by S-Ou**
Currently just with dummy data

### CODE REVIEW

File: src/contexts/TitleContext.tsx
Comment (pumbas600) [inline src/contexts/TitleContext.tsx]: You can simplify this type slightly :)
```suggestion
    // An orderId of -1 is reserved for the release order
	const [sequences, setSequences] = useState<Record<number, Sequence>>({});
```

**Review by pumbas600 — APPROVED** [review]
:)

I would hold off on merging this until your other two PRs are merged first though :)

File: src/app/title/page.tsx
Comment (S-Ou) [inline src/app/title/page.tsx:307]: would it be possible to do something like
```ts
... = useTitleContext({
  orderId = 1234,
  orderType = "sequential"
})
```
so it initialises with those values?

... that said, those params could also potentially have their own contexts or global values or whatever which would negate the need for this anyways

Comment (pumbas600) [reply src/app/title/page.tsx:307]: Unfortunately not :(

**Pull request merged**
